code/modules/mob/living/simple_animal/friendly/drone/drones_as_items.dm

```diff
@@ -19,6 +19,8 @@
 
 /obj/item/drone_shell/Initialize()
 	. = ..()
+	if(!CONFIG_GET(flag/drone_ghost_role))
+		qdel(src)
```

Comment:
This is still terrible, if drones aren't enabled drone shell dispensers shouldn't accept metal/glass. This shouldn't be a silent metal/glass sink.